v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/settings_manager.py

```diff
@@ -62,7 +63,16 @@ def __upgrade_obsolete_settings(self, settings: dict):
         }.items():
             value = settings.get(setting_key)
             if value == obsolete:
-                self.__log(f"Changed obsolete setting: {obsolete} to {new}.")
+                self.__log(f"Changed setting ({setting_key}): {obsolete} to {new}.")
+                settings.update({setting_key: new})
+
+        setting_key = "input_text.fm_host_url"
+        for obsolete, new in {
```

Comment:
Does it make sense to use a for-loop for this?
I would expect that for this one key, we only need to change it to one new value.

Reply:
Indeed, will change